web/Database/LandlordDbModel.cs

```diff
@@ -13,5 +13,8 @@ public class LandlordDbModel
     public string Email { get; set; } = null!;
     public string Phone { get; set; } = null!;
 
+    public string LandlordStatus { get; set; } = null!;
+    public bool CharterStatus { get; set; } = false;

```

Comment:
If this is provided by the landlord then we should make that explicit so it's not confused with the admin approval for charter. I'd call it e.g. LandlordProvidedCharterStatus